src/js/components/Skeleton/StyledSkeleton.js

```diff
@@ -0,0 +1,38 @@
+
+import styled from 'styled-components';
+
+import {
+  backgroundStyle,
+  edgeStyle,
+  genericStyles,
+  heightStyle,
+  roundStyle,
+  widthStyle,
+} from '../../utils';
+
+export const StyledSkeleton = styled.div`
```

Comment:
curious why `div` and not `box` ?

Reply:
partly because Box has some special interpretation when the skeleton context is present. I started that way but it got more complicated

Reply:
good to know figured there was some reasoning behind picking `div` over `box`

Reply:
Should we add a comment about this in the code? ^^